tests/smart_contracts/test_views.py

```diff
@@ -388,19 +389,21 @@ def test_cannot_generate_pdf_for_agreement_with_status_other_than_created(
                 name="contract_template.md",
             )
         )
+        assigner_user = UserFactory(organization=organization)
+        assignee_user = UserFactory(organization=agreement.assignee)
         assigner_representative = ContactFactory(
             dataset=dataset,
-            object_id=organization.pk,
-            content_type=ContentType.objects.get_for_model(organization),
-            email=organization.email,
-            phone=organization.phone,
+            object_id=assigner_user.pk,
+            content_type=ContentType.objects.get_for_model(User),
```

Comment:
```suggestion
            content_object=assigner_user
```
I suggest using this way of assigning `GenericForeignKey` relations everywhere

Reply:
Changes to the ContactFactory would have to be made if I wanted to do this, so I will be skipping this. Thanks for the suggestion, tho.